src/helpers/openAppLink.ts

```diff
@@ -0,0 +1,17 @@
+import EmailScreenParams from 'models/EmailScreenParams'
+import KetlPath from 'models/KetlPath'
+
+export default function ({
+  blank,
+  domain,
+  path,
+  token,
+}: EmailScreenParams & {
+  blank?: boolean
+  path: KetlPath
+}) {
+  const link = `ketl://${path}`
+  const query = `?domain=${domain}&token=${token}`
```

Comment:
I'm afraid that this will lead to errors if we try to redirect the user to the token page in the Ketl because you will do it with a link like `ketl://token?domain=undefined&token=SOME_TOKEN`
Recheck the work of this function, please